extensions/container-image/container-image-buildpack/deployment/src/main/java/io/quarkus/container/image/buildpack/deployment/BuildpackConfig.java

```diff
@@ -14,13 +14,13 @@ public class BuildpackConfig {
     /**
      * The buildpacks builder image to use when building the project in jvm mode.
      */
-    @ConfigItem
+    @ConfigItem(defaultValue = "paketocommunity/builder-ubi-base")
     public Optional<String> jvmBuilderImage;
```

Comment:
Given you provided a default value, you should drop the `Optional`.

Same for the one just below.

It's not mandatory so I suppose we can live with it if you don't have to push other changes but if CI doesn't pass and you do, please drop the Optional. Thanks!